_episodes/01-intro.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Python Fundamentals
+teaching: 20
+exercises: 10
+questions:
+- "What basic data types can I work with in Python?"
+- "How can I create a new variable in Python?"
+- "Can I change the value associated with a variable after I create it?"
+- "What is the order of mathematical operation in Python?"
```

Comment:
Fair point. I usually mention it after the first line of code. Because this is a split, I think the proper course of actions is to remove it now and add it later (if we decide that we need it).